js/main.js

```diff
@@ -1,2 +1,29 @@
 /* global data */
 /* exported data */
+
+var title = document.querySelector('#title');
+var photoUrl = document.querySelector('#photo-url');
+var notes = document.querySelector('#notes');
+var img = document.querySelector('img');
+var form = document.querySelector('form');
+
+function saveEntry(event) {
+  event.preventDefault();
+  var entry = {};
+  entry.title = title.value;
+  entry.photoUrl = photoUrl.value;
+  entry.notes = notes.value;
```

Comment:
Instead of selecting and saving these elements into individual variables on the global scope, I would just use the `elements` property on the `form` itself.